docs/source/concepts/index.rst

```diff
@@ -0,0 +1,222 @@
+
+.. image:: /_images/simple_zip_folder.png
+    :alt: Simple ZIP Folder Layout
+    :height: 540px
+    :align: center
+
+Metadata ZIP
+""""""""""""
+Images can be uploaded in ZIP file format, with a CSV manifest.
+This manifest, which must be named :code:`metadata.csv` and placed at the top-level of the zipped directory, contains mappings between relative filepaths and labels.
+The images in the ZIP can be in an arbitrary layout.
+
+The metadata file must be formatted as a standard CSV (e.g., use :code:`,` as the delimiter and :code:`"` as the quote character).
+
+.. image:: /_images/metadata_zip_folder.png
+    :alt: Metadata ZIP Folder Layout
+    :height: 540px
+    :align: center
+
+External Media
+""""""""""""""
+Images can be supplied using public URLs in any of our supported tabular formats (CSV, JSON, XLS/XLSX, DataFrame). If using a CSV, ensure that it is formatted as a standard CSV (e.g., use :code:`,` as the delimiter and :code:`"` as the quote character).
+If using JSON, ensure that it is encoded as a JSON array of JSON objects, where each object has the same set of keys.
+Values must be primitives: Cleanlab Studio doesn’t support nested JSON structures, you must flatten them.
+
+One of your columns should contain a sequence of URLs, each pointing to a single hosted image.
+These URLs must be either public or pre-signed; no additional authentication can be required to access the images.
+Your dataset can contain arbitrary other columns, in addition to the image and label columns.
+
+.. collapse:: Example dataset snippet
+    :open:
+
+    =  ==  ================================================= =====
+    _  id                                                img label
+    =  ==  ================================================= =====
+    0   0  https://s.cleanlab.ai/DCA_Competition_2023_Dat...     c
+    1   1  https://s.cleanlab.ai/DCA_Competition_2023_Dat...     h
+    2   2  https://s.cleanlab.ai/DCA_Competition_2023_Dat...     y
+    3   3  https://s.cleanlab.ai/DCA_Competition_2023_Dat...     p
+    4   4  https://s.cleanlab.ai/DCA_Competition_2023_Dat...     j
+    =  ==  ================================================= =====
+
+Schemas
+-------
+Schemas are used to define the data and feature types of the fields in your uploaded datasets.
+While Cleanlab Studio is generally quite accurate with inferring these types, sometimes you may want to override our inferences.
+
+Cleanlab Studio supports the following data and feature types:
+
+=========   =================================================
+Data type   Feature type
+=========   =================================================
+string      text, categorical, datetime, identifier, filepath
+integer     categorical, datetime, identifier, numeric
+float       datetime, numeric
+boolean     boolean
+=========   =================================================
+
+In the case that you want to override the inferred schema, you can pass in a schema override.
```

Comment:
Should maybe mention that this is for Python API (not CLI) since everything else in this section is concepts that apply to both